### ISSUE TITLE
Need the support for Anki 23.12

### ISSUE THREAD

**Issue by s609078902** [trgkanki/noctrlq#1]
I need this addon so much. It would be nice if the addon can support for Anki 23.12

**Comment by phu54321** [comment]
Doesn't it work?

**Comment by s609078902** [comment]
> Doesn't it work?

No. It showed the warning "The following add-ons failed to load: ⁨noctrlq Ask before exitingquitting CtrlQ AltF4⁩".
